tex/chapters/07-ApplyingModel.tex

```diff
@@ -0,0 +1,197 @@
+See Section \ref{sec:individual_strategies} for explanations of how these strategies operate.
+
+
+
+\section{Determining the sensitivity of the Model}
+
+All possible MemoryOne strategies are defined by the space $S = [0, 1]^4_{\mathbb{R}}$.
+For a vector $s \in S$ and a deviation $\delta \in (0, 1)$, 16 new deviated vectors are created by taking the sum of $s$ with all possible vectors $(\pm \delta, \pm \delta, \pm \delta, \pm \delta)$, but bounding each element in the range $[0, 1]$.
+For example, with $s = (0.2, 0, 0.5, 0.95)$ and $\delta = 0.1$, the 16 vectors produced are:
+
+\begin{itemize}
+\begin{multicols}{4}
+  \item $(0.3, 0.1, 0.6, 1.0)$
+  \item $(0.3, 0.1, 0.6, 0.85)$
+  \item $(0.3, 0.1, 0.4, 1.0)$
+  \item $(0.3, 0.1, 0.4, 0.85)$
+  \item $(0.3, 0.0, 0.6, 1.0)$
+  \item $(0.3, 0.0, 0.6, 0.85)$
+  \item $(0.3, 0.0, 0.4, 1.0)$
+  \item $(0.3, 0.0, 0.4, 0.85)$
+  \item $(0.1, 0.1, 0.6, 1.0)$
+  \item $(0.1, 0.1, 0.6, 0.85)$
+  \item $(0.1, 0.1, 0.4, 1.0)$
+  \item $(0.1, 0.1, 0.4, 0.85)$
+  \item $(0.1, 0.0, 0.6, 1.0)$
+  \item $(0.1, 0.0, 0.6, 0.85)$
+  \item $(0.1, 0.0, 0.4, 1.0)$
+  \item $(0.1, 0.0, 0.4, 0.85)$
+\end{multicols}
+\end{itemize}
+
+The 16 deviated vectors are then entered into an Ashlock tournament to produce the corresponding results table where all strategies are compared (as described in Section \ref{sec:ashlock_tourn}).
+The model is then be applied to this set of results, and the proportion of comparisons predicted correctly is recorded.
+As all the strategies have been defined by unique vectors, it is known that all the strategies are different and so the model should predict that they all different.
+By repeating this process for different vectors $s \in S$ whilst also varying $\delta$, Figure \ref{fig:m1_correct} is produced.
+
+\begin{figure}[htbp!]
+    \centering
+    \includegraphics[width=0.9\textwidth]{../img/ML/proportion-correct.png}
+    \caption{Performance of model for increasing $\delta$}
+    \label{fig:m1_correct}
+\end{figure}
+
+The model appears to reach a reasonable level of accuracy when $\delta > 0.3$.
+However, there is a large amount of variance for all $\delta$ used.
+This suggests that the original vector has a significant effect on how the model performs.
+For some MemoryOne strategies even very small deviations will be noticed by the model, but for other MemoryOne strategies it takes a much larger change before the model will recognise the difference.
+
+\section{Applying Model to Axelrod-Python}
+
+The final section of this report addresses the problem of determining when strategies in Axelrod-Python are identical.
+The assumption made in Section \ref{sec:training_model} must now be ignored.
+It stated: \textit{Strategies with different names are assumed to be different}, but this is not necessarily the case.
```

Comment:
`It was assumed that... but this is ultimately not known, whilst all the implementations are different it is possible that there are equivalence of strategies.`